macs2.wdl

```diff
@@ -64,6 +78,13 @@ task PeakCalling {
         sampleName: {description: "Name of the sample to be analysed", category: "required"}
         outDir: {description: "All output files will be written in this directory.", category: "advanced"}
         nomodel: {description: "Whether or not to build the shifting model.", category: "advanced"}
+        gensz: {description: "macs2 argument for setting the mappable genome size or effective genome size which is defined as the genome size which can be sequenced."}
+        pval_thres: {description: "macs2 argument for setting the p-value cutoff. If -p is specified, MACS2 will use p-value instead of q-value."}
+        shiftsize: {description: "macs2 argument to set an arbitrary shift in bp. Can be negative to indicate direction"}
+        extsize: {description: "macs2 argument to extend reads in 5'->3' direction to fix-sized fragments."}
+        bdg: {description: "macs2 argument that ebanbles the storage of the fragment pileup, control lambda in bedGraph files."}
+        keepdup: {description: "macs2 argument that controls the behavior towards duplicate tags at the exact same location."}
+        callsummits: {description: "macs2 argument to reanalyze the shape of signal profile to deconvolve subpeaks within each peak called from the general procedure."}
```

Comment:
```suggestion
        gensz: {description: "macs2 argument for setting the mappable genome size or effective genome size which is defined as the genome size which can be sequenced.", category: "advanced"}
        pval_thres: {description: "macs2 argument for setting the p-value cutoff. If -p is specified, MACS2 will use p-value instead of q-value.", category: "advanced"}
        shiftsize: {description: "macs2 argument to set an arbitrary shift in bp. Can be negative to indicate direction.", category: "advanced"}
        extsize: {description: "macs2 argument to extend reads in 5'->3' direction to fix-sized fragments.", category: "advanced"}
        bdg: {description: "macs2 argument that enables the storage of the fragment pileup, control lambda in bedGraph files.", category: "advanced"}
        keepdup: {description: "macs2 argument that controls the behavior towards duplicate tags at the exact same location.", category: "advanced"}
        callsummits: {description: "macs2 argument to reanalyze the shape of signal profile to deconvolve subpeaks within each peak called from the general procedure.", category: "advanced"}
```